<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>muroaguiClass</class>
 <widget class="QMainWindow" name="muroaguiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Muroa Gui</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>current artist - current title</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QSlider" name="posSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="progressLabel">
      <property name="text">
       <string>00:00 / 00:00</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="CCollectionView" name="collectionView"/>
      <widget class="CPlaylistView" name="playlistView"/>
      <widget class="CPlaylistView" name="nextToPlayView"/>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="stopBtn">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="MuroaGui.qrc">
        <normaloff>:/icons/icons/media-playback-stop.png</normaloff>:/icons/icons/media-playback-stop.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QToolButton" name="playBtn">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="MuroaGui.qrc">
        <normaloff>:/icons/icons/media-playback-start.png</normaloff>:/icons/icons/media-playback-start.png</iconset>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>935</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Connection">
    <property name="title">
     <string>&amp;Connection</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionPlayPause"/>
    <addaction name="actionStop"/>
   </widget>
   <widget class="QMenu" name="menuCo_llection">
    <property name="title">
     <string>Co&amp;llection</string>
    </property>
    <addaction name="actionGet_Revision_1"/>
    <addaction name="actionGet_next_Revision"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Connection"/>
   <addaction name="menuCo_llection"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionGet_Revision_1">
   <property name="text">
    <string>get Revision 1</string>
   </property>
  </action>
  <action name="actionGet_next_Revision">
   <property name="text">
    <string>get next Revision</string>
   </property>
  </action>
  <action name="actionPlayPause">
   <property name="icon">
    <iconset resource="MuroaGui.qrc">
     <normaloff>:/icons/icons/media-playback-start.png</normaloff>:/icons/icons/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Play/Pause</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="MuroaGui.qrc">
     <normaloff>:/icons/icons/media-playback-stop.png</normaloff>:/icons/icons/media-playback-stop.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPlaylistView</class>
   <extends>QListView</extends>
   <header>CPlaylistView.h</header>
  </customwidget>
  <customwidget>
   <class>CCollectionView</class>
   <extends>QTreeView</extends>
   <header>CCollectionView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MuroaGui.qrc"/>
 </resources>
 <connections/>
</ui>
